packages/remix/src/utils/utils.ts

```diff
@@ -73,3 +79,10 @@ export const getEnvVariable = (name: string, context: AppLoadContext | undefined
 
   return '';
 };
+
+export const getSpaMode = (): boolean => {
```

Comment:
```suggestion
export const inSpaMode = (): boolean => {
```
❓  Would `inSpaMode` makes more sense than `getSpaMode`?